backend/src/ee/services/audit-log/audit-log-types.ts

```diff
@@ -2458,6 +2463,27 @@ interface AutomatedRenewPkiSubscriberCert {
   };
 }
 
+interface AutomatedRenewCertificate {
+  type: EventType.AUTOMATED_RENEW_CERTIFICATE;
```

Comment:
IGNORE PLACEMENT OF COMMENT

For ineligible ones, I think we should show a status and maybe explain why it's ineligible? Just to keep it consistent 

<img width="1509" height="631" alt="image" src="https://github.com/user-attachments/assets/06d9c040-68fe-4fd6-a723-b34914636c62" />